leader-election-impl/src/main/java/com/palantir/paxos/PaxosLatestRoundVerifierImpl.java

```diff
@@ -51,7 +55,12 @@ public PaxosQuorumStatus isLatestRound(long round) {
     }
 
     private boolean acceptorAgreesIsLatestRound(PaxosAcceptor acceptor, long round) {
-        return round >= acceptor.getLatestSequencePreparedOrAccepted();
+        try {
+            return round >= acceptor.getLatestSequencePreparedOrAccepted();
+        } catch (Exception e) {
+            log.debug("latest sequence retrieval failed for acceptor: {}", acceptor, e);
```

Comment:
You probably want this to be a higher level - internally most things aren't logged at DEBUG. I realise that if a node is down this could cause a bit of spam, but INFO is probably okay.